### PR TITLE
Feat: Add postrm in dummy deb

### PR DESCRIPTION
**Pull request by rain1598**
# Purpose

Pacstall dummy packages currently cannot be removed with apt

# Approach

Adds a postrm file that calls pacstall

### CODE REVIEW

**Comment by Henryws** [comment]
```bash
[+] INFO: Creating dummy package
dpkg-deb: error: parsing file '/tmp/pacstall/bat-pacstall/DEBIAN/control' near line 8 package 'bat-pacstall':
 'Conflicts' field, syntax error after reference to package 'bat'
[!] ERROR: Couldn't create dummy package
[!] ERROR: Failed to install bat
➜  ~
```

**Comment by Henryws** [comment]
```bash
[+] INFO: Creating dummy package
dpkg-deb: error: maintainer script 'postrm' has bad permissions 644 (must be >=0555 and <=0775)
[!] ERROR: Couldn't create dummy package
[!] ERROR: Failed to install clib
```

**Review by D-Brox — APPROVED** [review]

File: misc/scripts/install-local.sh [hunk 1]
```diff
@@ -130,7 +130,7 @@ Version: $version\n"| sudo tee "$SRCDIR/$name-pacstall/DEBIAN/control" > /dev/nu
 		printf "\n" | sudo tee -a "$SRCDIR/$name-pacstall/DEBIAN/control" > /dev/null
 	fi
 	printf "Architecture: all
-Essential: yes
+Essential: no
 Section: Pacstall
 Priority: optional\n" | sudo tee -a "$SRCDIR/$name-pacstall/DEBIAN/control" > /dev/null
 	if [[ -v replace ]]; then
```
Comment (D-Brox) [inline misc/scripts/install-local.sh:136]: `if [[ -n $replace ]]` should probably fix the first issue TB pointed out

**Review by D-Brox — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
new bug
```bash
[+] INFO: Creating dummy package
[+] INFO: clib has optional dependencies that can enhance its functionalities
Optional dependencies:

Do you want to install them [Y/n]
```

**Comment by Henryws** [comment]
```bash
[+] INFO: Creating dummy package
/var/lib/dpkg/info/clib-pacstall.postrm: 1: [[: not found
[+] INFO: Installing dependencies
/var/lib/dpkg/info/clib-pacstall.postrm: 1: [[: not found
[+] INFO: Retrieving packages
Initializing download: https://github.com/clibs/clib/archive/refs/tags/2.7.0.tar.gz
File size: unavailable
Opening output file 2.7.0.tar.gz
Server unsupported, starting from scratch with one connection.
Starting download

Connection 0 finished

Downloaded 171.974 Kilobyte(s) in 0 second(s). (423.74 KB/s)
[+] INFO: Running prepare in fakeroot. Do not enter password if prompted
[+] INFO: Running build in fakeroot. Do not enter password if prompted
make: *** No targets specified and no makefile found.  Stop.
[+] INFO: Installing
make: *** No rule to make target 'install'.  Stop.
install: cannot stat 'LICENSE': No such file or directory
[+] INFO: Symlinking files
[+] INFO: Storing pacscript
[+] INFO: Cleaning up
```

**Review by D-Brox — APPROVED** [review]

**Comment by D-Brox** [comment]
I forgot to test `-git`

**Pull request merged**